playbooks/tasks/portal-docker-services-start.yml

```diff
@@ -74,7 +80,9 @@
 
 # Prune docker
 - name: Prune docker
-  command: docker system prune
+  ansible.builtin.command: docker system prune --force
```

Comment:
If we are pruning before we start the services, is this doing anything?

Reply:
If no docker container/image/network was run/pulled/built/set on this machine, it's doing nothing,
otherwise it's pruning things from previous docker executions (e.g. from previous deployments).